fhir-server/src/main/java/org/linuxforhealth/fhir/server/filter/rest/FHIRRestServletFilter.java

```diff
@@ -135,7 +138,11 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
             FHIRConfiguration.validateTenantId(tenantId);
             FHIRConfiguration.validateDatastoreId(dsId);
 
-            // Checks for Valid Tenant Configuration
+            // Check if the tenant Id is present in the list of configured tenants and return the tenant Id from the configured tenant list.
+            // This is done to avoid Uncontrolled user input data used in path expression
+            
+            tenantId = fetchValidConfiguredTenantId(tenantId);
+            
             checkValidTenantConfiguration(tenantId);
```

Comment:
Can this validation be removed now?

Reply:
yes, I think we can remove this now that you've implemented fetchValidConfiguredTenantId instead